rune/init.go

```diff
@@ -45,6 +45,8 @@ var initCommand = cli.Command{
 			// or write it to stderr because the parent process will handle this
 			os.Exit(1)
 		}
-		panic("libcontainer: container init failed to exec")
+		os.Exit(0)
+		// make compiler happy
+		return nil
```

Comment:
You need to do this under the context of a libenclave function rather than such a common place.